libraries/standard/http/lexicon.txt

```diff
@@ -0,0 +1,144 @@
+absolutevalue
+addrangeheader
+addrangerequest
+aggregator
+api
+ascii
+belown
```

Comment:
Shouldn't be supported?

Reply:
This was fixed but missed from deleting thanks for catching